spec/index.bs

```diff
@@ -2409,7 +2409,7 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
 
         <dfn>binding validation</dfn>: Each |bindingDescriptor|.{{GPUBindGroupLayoutEntry/binding}} in |descriptor| must be unique.
 
-        <dfn>vertex shader binding validation</dfn>: {{GPUBindingType/storage-buffer}} is not allowed.
+        <dfn>vertex shader binding validation</dfn>: {{GPUBindingType/storage-buffer}} and {{GPUBindingType/writeonly-storage-texture}} are not allowed.
```

Comment:
ok, this is actually quite worrying to me. This validation rule today (even outside of the PR) doesn't allow storage buffers in the vertex stage. This is too strict of a limitation. What we should disallow is the *writable* storage buffers.

Reply:
Well, we have a separate binding type: readonly-storage-buffer. And readonly-storage-buffer is allowed in vertex stage.

Reply:
ah right, totally forgot about this one, phew